integration/shareddata/composites.go

```diff
@@ -73,6 +73,33 @@ var Composites = &Values[string]{
 	},
 }
 
+// Mixed contains composite and scalar values for tests. It is used for sorting
+// mixture of array and scalar documents.
+var Mixed = &Values[string]{
+	name:     "Mixed",
+	backends: []string{"ferretdb-pg", "mongodb"},
+	data: map[string]any{
+		"null":        nil,
+		"unset":       unset,
+		"array-empty": bson.A{},
+		"array-null":  bson.A{nil},
+	},
+}
+
+// ArrayAndDocuments contain composite values for tests.
+var ArrayAndDocuments = &Values[string]{
```

Comment:
@noisersup you caught a bug :hugs: The commit relevant for this fix was https://github.com/FerretDB/FerretDB/pull/2263/commits/ecf2b70c7937f7711609fd350ae31656331f6629